test/integration/coscheduling_test.go

```diff
@@ -408,6 +442,10 @@ func makeCRD() *apiextensionsv1.CustomResourceDefinition {
 										Type:    "integer",
 										Minimum: &min,
 									},
+									"minResources": {
+										Type:                   "object",
+										XPreserveUnknownFields: &preserveUnknownFields,
```

Comment:
What's the intention here?

Reply:
I found in apiextension/v1, minResources would be ignored if we do not set XPreserveUnknownFields, but in apiextension/v1beta1, it works

Reply:
I found in apiextension/v1beta1, default value of this field is true

Reply:
https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/v1beta1/defaults.go#L73

Reply:
If I understand  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types.go#L65-L71 cleary, it's good enough to just add `minResources` to OpenAPI schema, rather than additionally defining "XPreserveUnknownFields", right?

Reply:
PR changed

Have changed `minResources`  to `AdditionalPropertities`

See https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/validation/validation.go#L756